### PR TITLE
Release v6.0.4

### PR DESCRIPTION
**Pull request by mgrassotti**
Bump version to 6.0.4. Details in CHANGELOG.md.

Related to https://github.com/alphagov/govuk_frontend_toolkit_gem/pull/15

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 6.0.4
+
+- Avoid the dependency from ActionCable, introduced in Rails 5. Related to https://github.com/alphagov/govuk_frontend_toolkit_gem/pull/15
```
Comment (cbaines) [inline CHANGELOG.md]: I think here, you should just mention that there are no changes, and that this release is being done to release changes in the corresponding Gem.

Talking about anything Ruby or Rails related is a bit out of context, and for example, isn't applicable if you are using the Node module, or some other release.

**Comment by mgrassotti** [comment]
@cbaines 👍  done

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 6.0.4
+
+- Version bumping related to https://github.com/alphagov/govuk_frontend_toolkit_gem/pull/15
```
Comment (cbaines) [inline CHANGELOG.md]: I think the content here is fine now. There seems to be a style for referring to Pull Requests though, so I'd change this to follow that, see the `([PR #15](...` like links in the rest of the changelog.

Comment (mgrassotti) [inline CHANGELOG.md]: Yes I saw them, but these are related to this repo, so I thought leaving the link should highlight that this refers the gem repo. I will change it to make it a little nicer maybe.

Comment (mgrassotti) [inline CHANGELOG.md]: @cbaines 👍  done

Comment (cbaines) [inline CHANGELOG.md]: Great

**Pull request merged**